site/src/server.rs

```diff
@@ -403,16 +403,12 @@ async fn serve_req(server: Server, req: Request) -> Result<Response, ServerError
         "/perf/self-profile-raw" => Ok(to_response(
             request_handlers::handle_self_profile_raw(check!(parse_body(&body)), &ctxt).await,
         )),
-        "/perf/graph" => Ok(
-            match request_handlers::handle_graph(check!(parse_body(&body)), &ctxt).await {
+        "/perf/graphs" => Ok(
+            match request_handlers::handle_graphs(check!(parse_body(&body)), &ctxt).await {
                 Ok(result) => {
-                    let mut response = http::Response::builder()
+                    let response = http::Response::builder()
                         .header_typed(ContentType::json())
                         .header_typed(CacheControl::new().with_no_cache().with_no_store());
-                    response.headers_mut().unwrap().insert(
-                        hyper::header::ACCESS_CONTROL_ALLOW_ORIGIN,
-                        hyper::header::HeaderValue::from_static("*"),
-                    );
```

Comment:
Looks like @leeoniya was using it for testing the new graph implementation -- https://github.com/rust-lang/rustc-perf/issues/742#issuecomment-670339622.

The header is also set on success in `run_server` anyway, so I don't think having it here matters. Could remove it from `/perf/bootstrap` endpoint similarly.

If we want to remove this from `run_server` too, then it might make sense to go back to returning a msgpack format (using `to_response`, updating front end accordingly).

Reply:
I removed this because the header is being set elsewhere.